rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```
@@ -487,6 +487,24 @@ int __pal_exec(char *path, char *argv[], pal_stdio_fds *stdio, int *exit_code)
 	return 0;
 }
 
+int __pal_get_local_report(void *targetinfo, void *report)
+{
+	uint8_t report_data[64] = { 0, };
+	struct sgx_report report_align;
+	int ret;
+
+	ret = SGX_ENTER_3_ARGS(ECALL_REPORT, (void *)secs.base, targetinfo,
+						report_data, &report_align);
+	if (ret)
+		fprintf(stderr, "failed to get report\n");
```

Return ret if failed to get local report, rather than still returning 0.